src/App.js

```diff
@@ -1,24 +1,21 @@
-import logo from './logo.svg';
-import './App.css';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+} from 'react-router-dom';
+import BookList from './components/BookList/BookList';
+import Categories from './components/Categories/Categories';
+import Header from './components/Header/Header';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Router>
+      <Header />
+      <Switch>
+        <Route path="/" exact component={BookList} />
```

Comment:
* Kindly create a component that Renders a form with inputs for the title and the author of the book with a submit button but the button isn't expected to work for now just needs to be rendered
* Refer to project sneak peak 
![image](https://user-images.githubusercontent.com/52497246/130642369-5148d970-6ade-42fa-bec7-49c68f8482d7.png)

Reply:
This is a requirement that I have not met and I will submit for the next review